keyboards/bm40hsrgb/keymaps/sigma-squared/rgb_matrix_user.inc

```diff
@@ -0,0 +1,440 @@
+// !!! DO NOT ADD #pragma once !!! //
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
This is the only remaining issue that I see.

Reply:
This is the only remaining issue that I see.